packages/locales/lib/human/de.json

```diff
@@ -734,5 +741,21 @@
   "reset_s2cells": "S2 Zellen zurücksetzen",
   "reset_spawnpoints": "Spawnpoints zurücksetzen",
   "reset_submission_cells": "Submission Zellen zurücksetzen",
-  "hisuian": "Hisui"
+  "hisuian": "Hisui",
+  "spacial_rend_range": "Raumschlag Effekt Bereich",
+  "key": "Schlüssel",
+  "ai": "KI",
+  "human": "Human",
+  "locales": "Lokalisierung",
+  "instructions": "Anleitung",
+  "locale_instructions_1": "Wählen im Dropdown-Menü eine Sprache aus.",
+  "locale_instructions_2": "Gib in der Spalte \"Human\" die gewünschten Übersetzungen ein.",
+  "locale_instructions_3": "Klicke auf die Schaltfläche \"$t(download)\", um eine JSON-Datei herunterzuladen.",
+  "locale_instructions_4": "Fork den GitHub-Repo-Link unten",
+  "locale_instructions_5": "Erstelle einen neuen Branch und benennen ihn mit der Sprache, die übersetzt werden soll.",
+  "locale_instructions_6": "Ersetze den Inhalt von \"packages/locales/lib/human/{{lng}}.json\" durch die heruntergeladene Datei.",
+  "locale_instructions_7": "Erstellen eine neue Pull-Anfrage auf den \"develop\" Branch.",
```

Comment:
It's common to use "Pull Request" in german, even Github itself shows "Pull-Request", not "Anfrage" :)